### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by jb822f** [SSAgov/ANDI#196]
It would be great if ANDI could check for redundant/duplicative text in accessible names or caused by alt text. Below is a common example: 

<a href="#"><img alt="Wireless" src="images/wls.png">
  Wireless
</a>

and/or

<img alt="Wireless" src="images/wls.png">
<a href="#">Wireless</a>

The WAVE tool and axe DevTools both detect it. ANDI reports the output for the link as WirelesWireless but does not list it in a list of potential issues. Thanks!

**Comment by JohnCotterSSA** [comment]
I can understand how the first example has some problematic verbosity, however in the second example, those elements are not programmatically associated with each other. Do the tools you mention flag the second example as a problem?

**Comment by jb822f** [comment]
Yes, and the warning message from the WAVE tool is the same for both code examples, "Redundant alternative text: The alternative text for an image is the same as nearby or adjacent text." Axe does not detect that second instance, but it should flag it for confirmation. Thank you for replying and considering this.
